src/components/Participants/Participants.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react"; 
+import style from './Participants.module.css';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+class Participants extends Component {
+  constructor(props) {
+    super(props)
+    
+    this.state = {
+      host: "",
+      email: "",
+      participants: [{ name: ""}],
```

Comment:
host can be name, email_address pair in state?
paricipants could be name, email_address pairs in state?